packages/xo-web/src/xo-app/jobs/overview/index.js

```diff
@@ -100,46 +100,18 @@ const ACTIONS = [
 // ===================================================================
 
 @addSubscriptions({
-  users: subscribeUsers,
+  jobs: [cb => subscribeJobs(jobs => cb(keyBy(jobs, 'id'))), {}],
+  schedules: [subscribeSchedules, []],
+  users: [subscribeUsers, []],
```

Comment:
Was it necessary to also set a default value for `users`?

Reply:
Indeed it's not as `filter` will handle `undefined`, but it will be replaced by the next commit anyway.

Do you want me to change this?

Reply:
No, you're right, it doesn't matter since it becomes necessary in the next commit anyway. I was just reading commit by commit and was just wondering.